content/v3/pulls/comments.md

```diff
@@ -66,8 +66,13 @@ Name | Type | Description
 `path`|`string` | **Required**. The relative path of the file to comment on.
 `position`|`integer` | **Required**. The line index in the diff to comment on.
 
-Note: When passing the `commit_id`, use the SHA of the latest commit in the pull request or your comment may appear as "outdated" if the specified `position` has been modified in a subsequent commit.
+{{#tip}}
 
+When passing the `commit_id`, use the SHA of the latest commit in the pull request or your comment may appear as "outdated" if the specified `position` has been modified in a subsequent commit.
+
+To comment on a specific line, in a file, you will need to first determine the position, in the diff. GitHub offers a `application/vnd.github.v3.diff` media type which you can use in a preceding request to view the pull request's diff. The diff needs to be [interpreted](https://en.wikipedia.org/wiki/Diff_utility#Unified_format) to translate *line in file* to *position in diff*. The `position` value is the number of lines down from the first "@@" hunk header in the file you would like to comment on. The line just below the "@@" line is position 1, the next line is position 2, and so on. The position in the file's diff continues to increase through lines of whitespace and additional hunks until a new file is reached.
```

Comment:
Sorry, I found just a couple of nitpicks… There are a couple commas here that aren't necessary; could we get rid of the following commas?

* "To comment on a specific line, in a file"
* "you will need to first determine the `position`, in the diff"

Also, I noticed a couple things to help match this more with the style of the rest of our documentation:

* Changing "to translate *line in file*" to "to translate from the *line in the file*"
* Changing "to *position in diff*" to "to a *position in the diff*"

That should be it 😅